ivre/db/mongo.py

```diff
@@ -1127,6 +1127,24 @@ def searchports(self, ports, protocol='tcp', state='open', neg=False):
             for port in ports]}}
 
     @staticmethod
+    def searchobjectid(oid, neg=False):
+        """Filters records by their ObjectID.
+        `oid` can be a single or many (as a list or any iterable) object ID(s),
+        specified as strings or an `ObjectID`s."""
+        if isinstance(oid, basestring) or isinstance(oid, bson.objectid.ObjectId):
```

Comment:
You are going to hate me but I had a doubt and checked: there is a shorter (and hence better, as @commial usually says) way to write the test: `if isinstance(oid, (basestring, bson.objectid.ObjectId)):`.